lib/Launcher.js

```diff
@@ -90,8 +92,11 @@ class Launcher {
       chromeProcess.once('close', () => {
         // Cleanup as processes exit.
         if (temporaryUserDataDir)
-          removeSync(temporaryUserDataDir);
-        fulfill();
+          rimrafAsync(temporaryUserDataDir)
+            .then(() => fulfill())
+            .catch(err => console.error(err));
```

Comment:
let's reject the `waitForChromeToClose` promise

Reply:
In many cases we don't catch errors from this promise, so we can get an unhandledRejection.